packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx

```diff
@@ -66,7 +66,7 @@ export interface DashboardLayoutSlots {
    * Optional footer component used in the layout sidebar.
    * @default null
    */
-  sidebarFooter?: React.JSXElementConstructor<SidebarFooterProps>;
+  sidebarFooter?: React.ElementType;
```

Comment:
So the `SignInPage` slots are typed as `React.JSXElementConstructor<Props>` but the slots in `Account` are `React.ElementType`?
Shouldn't it be consistent all around? What do the other MUI component use?

Also the `SidebarFooterProps` have a `mini` boolean prop that can be useful, so we need the types to be aware of it...

Reply:
I've updated the `sidebarFooter` type; agree on the need for consistency - we can probably discuss this and raise a PR separately to bring consistency across components